index.html

```diff
@@ -150,5 +150,28 @@ <h2 class="list-of-skills">Skills</h2>
       </div>
     </div>
   </section>
+  <section class="contact-section">
+    <div class="contact-details">
+      <h2 class="paragraph_1">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h2>
+    </div>
+    <form class="form" action="">
+      <ul>
+        <li><input type="text" placeholder="Full Name"></li><hr>
+        <li><input type="email" placeholder="Email address"></li><hr>
```

Comment:
:orange_circle: [OPTIONAL] Please utilize CSS to create a bottom border on the input instead of using the HTML `<hr>` tag. If this was meant to be an horizontal divider, then your approach would be appropriate, but in this case it is just for styling the bottom of the input. You can implement it as follow 
```
input {
  border-bottom: 1px solid #000
}
```

Reply:
Thank you so much for the hint